src/api/app/models/decision.rb

```diff
@@ -24,6 +27,10 @@ def create_event
   def event_parameters
     { id: id, moderator_id: moderator.id, reason: reason, reportable_type: reports.first.reportable.class.name }
   end
+
+  def track_decision
+    RabbitmqBus.send_to_bus('metrics', "decision,kind=#{kind},hours_before_decision=#{((updated_at - created_at) / 1.hour).floor} count=1")
```

Comment:
`hours_before_decision` is a tag here - and tags don't like to have a large number of different values. Better add it as a value after the space.